From 36a728d4a4a4d563162cfe4dba93d290bee30267 Mon Sep 17 00:00:00 2001
From: Matthew Gerlach <matthew.gerlach@intel.com>
Date: Thu, 15 Sep 2016 11:47:21 -0700
Subject: [PATCH 019/151] FogBugz #403389-1: Altera Partial Reconfiguration IP
 Core

commit  771026af51f3d2d6b1025d797686c6191eddb8ed from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Add device tree bindings documentation for the Altera
Partial Reconfiguration IP Core.

Signed-off-by: Matthew Gerlach <matthew.gerlach@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 .../bindings/fpga/altera-partial-reconfig.txt        | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/fpga/altera-partial-reconfig.txt

diff --git a/Documentation/devicetree/bindings/fpga/altera-partial-reconfig.txt b/Documentation/devicetree/bindings/fpga/altera-partial-reconfig.txt
new file mode 100644
index 000000000000..bbbb9cdb3da7
--- /dev/null
+++ b/Documentation/devicetree/bindings/fpga/altera-partial-reconfig.txt
@@ -0,0 +1,12 @@
+Altera Partial Reconfiguration IP Core
+
+Required properties:
+- compatible : should contain "altr,pr-ip-core"
+- reg        : base address and size for memory mapped io.
+
+Example:
+
+	fpga_mgr: fpga-mgr@ff20c000 {
+		compatible = "altr,pr-ip-core";
+		reg = <0xff20c000 0x8>;
+	};
-- 
2.26.1

